frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -88,6 +141,37 @@ pods:
           export KAFKA_OPTS="-Dsun.security.krb5.debug=true $KAFKA_OPTS"
         {{/TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
       {{/TASKCFG_ALL_SECURITY_KERBEROS_ENABLED}}
+      {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+          export SECURE_JMX_KEY_STORE_KEY=`cat $MESOS_SANDBOX/jmx/key_store_pass`
+          export KEY_STORE_PATH=$MESOS_SANDBOX/jmx/key_store
+          if [ -f "$MESOS_SANDBOX/jmx/trust_store" ]; then
+              # load user provided trust store
+              export TRUST_STORE_PATH=$MESOS_SANDBOX/jmx/trust_store
+              export SECURE_JMX_TRUST_STORE_KEY=`cat $MESOS_SANDBOX/jmx/trust_store_pass`
+          else
+              # load user provided keystore certs to default truststore
+              export TRUST_STORE_PATH=$JAVA_HOME/lib/security/cacerts
+              export SECURE_JMX_TRUST_STORE_KEY="changeit"
+              self_signed_aliases=$(${JAVA_HOME}/bin/keytool -list -v -keystore $KEY_STORE_PATH -storepass $SECURE_JMX_KEY_STORE_KEY | grep 'Alias name:' | awk '{print $3}')
```

Comment:
I'm worried this might be fragile, for a few reasons:
1. `-v` is [documented](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) to be a human-readable, not machine-readable format, so it might change in a newer release or implementation of JDK.
1. we are running this on user-supplied data, and the fact that we run this in a pipeline will cause `bash` to ignore failure from `keytool` if any, and the next command will probably fail in a confusing way,
1. what if any of the aliases contain a space? Then `awk` will only return the first piece, and the next keytool command will fail miserably. Might be useful to add a test for this case, BTW.

Do you think we could just omit `-srcalias` and `-destalias` to have the command copy everything? Otherwise, we might need to write a little non-shell (Go?) program to do this, to be safe from embedded whitespace, otherwise this command might get ugly soon.

Reply:
> Do you think we could just omit -srcalias and -destalias to have the command copy everything?
let me run the tests with this

Reply:
it worked so I've removed all this